pkg/tasks/certs.go

```diff
@@ -31,3 +42,99 @@ func deployCAOnNode(s *state.State, node *kubeoneapi.HostConfig, conn ssh.Connec
 	s.Logger.Infoln("Uploading PKI files...")
 	return s.Configuration.UploadTo(conn, s.WorkDir)
 }
+
+func renewCerts(s *state.State) error {
+	if !s.ForceUpgrade {
+		s.Logger.Warn("Your control-plane certificates are about to expire in less then 90 days")
+		return nil
+	}
+	s.Logger.Warn("Your control-plane certificates are about to expire in less then 90 days")
+	s.Logger.Warn("Forced renew")
```

Comment:
Let's make this message more explicit, such as:
```suggestion
	s.Logger.Warn("Force renewing Kubernetes certificates")
```

Reply:
done